src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -51,9 +51,15 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             }
         }
 
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
-
+        if (!player.isOp())
+        {
+        	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
+            player.setOp(true);

```

Comment:
Formatting is out here, I dont know which one it should be just by looking, but line 56 and 57 are different and should not be.